tests/unit/drivers/test_cache_content_driver.py

```diff
@@ -0,0 +1,69 @@
+from pathlib import Path
+from typing import Any
+
+import numpy as np
+import pytest
+
+from jina import DocumentSet
+from jina.drivers.cache import BaseCacheDriver, CONTENT_HASH_KEY
+from jina.executors.indexers.cache import DocIDCache
+from jina.proto import jina_pb2
+from jina.types.document import uid, Document, bytes2id
+from tests import random_docs
+from tests.unit.drivers.test_cache_id_driver import MockCacheDriver
```

Comment:
avoid importing from different tests. I actually do not understand why are we having a different test file if the class is the same?